### ISSUE TITLE
Week 9 feedback

### ISSUE THREAD

**Issue by triashap5** [ShihChiangLo/comp-lab-class#1]
All the steps for generating output results were present but I could not find any file or notebook containing the analyses and figures. Is there anything that I am missing out on, or have you not completed the assignment yet/are you facing some issues with uploading the results?

**Comment by triashap5** [comment]
Okay, I checked the other branch. Everything seems perfect. All data files have been generated and the analyses are okay. However, I could not find the plot for diffusion times, as was asked for in the assignment. The plot for diffusion times vs inverse of Temperature needs to be plotted. Else every other thing is perfectly presented. 
Well done!

**Comment by ShihChiangLo** [comment]
Oh~ I guess the plot you are looking for is the one I named it after this"plot of MSD equals to one at different temperature.pdf". You can check whether it is the diffusion plot you mean. Sorry for the confusing file name

**Comment by triashap5** [comment]
Yeah I checked that! It's alright! The plot is alright! Good work!
